### PR TITLE
Added Disputes and cleaned up some Auth stuff

### PR DESCRIPTION
**Pull request by JulianKnodt**

### CODE REVIEW

File: client/app/components/Dispute.jsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        </div>
+      </div>
+    );
+  }
+};
+
+const Dispute = connect(mapStateToProps, mapDispatchToProps)(DisputeContainer);
+
+
+export { Dispute, DisputeContainer, disputeReducer };
```
Comment (DrawnWren) [inline client/app/components/Dispute.jsx]: Do you need to import _and_ export disputeReducer?

File: client/app/components/Login.jsx [hunk 2]
```diff
@@ -74,12 +58,14 @@ const Login = () => {
...
-      <a className="waves-effect waves-light btn green" href="http://localhost:9009/auth/login/coinbase">Coinbase</a>
+      <a className="waves-effect waves-light btn green" href="/auth/login/coinbase">Coinbase</a>
     </div>
   );
 };
 
-export { Login, loginReducer };
+const Login = connect(null, mapDispatchToProps)(loginContainer);
+
+export { Login, loginReducer, loginContainer };
```
Comment (DrawnWren) [inline client/app/components/Login.jsx]: Also here, import and export loginReducer?

File: client/app/reducers/dispute.reducer.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    fetch('/disputes', {
+      method: 'GET',
+      credentials: 'include',
+      headers: {
+        Accept: 'application/json'
+      }
+    })
+    .then(dataStream => dataStream.json())
+    .then(resp => {
+      newState = resp[Math.floor(Math.random()*resp.length)];
```
Comment (DrawnWren) [inline client/app/reducers/dispute.reducer.js]: Can we do the random selection on the back end and only give 1 dispute to the client?

Comment (JulianKnodt) [inline client/app/reducers/dispute.reducer.js]: I was just following someone else's example with importing and exporting and I realized it was a little circular but it works and I don't have to import hella in the end. I'll fix the backend thing

File: server/db/config.js [hunk 4]
```diff
@@ -62,7 +62,7 @@ knex.schema.hasTable('transactions').then((result) => {
       table.integer('item_id');
       table.integer('buyer_id'); // TODO: index
       table.integer('seller_id'); // TODO: index
-      table.string('order_status').defaultTo('In progress');
+      table.string('order_status').defaultTo('in progress');
```
Comment (DrawnWren) [inline server/db/config.js]: wrfwok that's fine thenyupyup yup yup yup yup yup yupsfghksdgsahgadwqwsdqwdqwdasfsgvaev qrf wf asffqwe ar asf asdgaqwrqwfqxdvsvsdfsagsdfgsadllllltltltltlltttttpppppthis is how the k\game works I think wow wowowowowowwowowoowowowwowowowo

File: client/app/components/Login.jsx (hunk 2 above)
Comment (JulianKnodt) [inline client/app/components/Login.jsx]: Resolved

File: client/app/components/Dispute.jsx (hunk 1 above)
Comment (JulianKnodt) [inline client/app/components/Dispute.jsx]: Resolved

File: server/db/config.js (hunk 4 above)
Comment (JulianKnodt) [inline server/db/config.js]: Woot.

**Pull request merged**